goserver/server/server.go

```diff
@@ -37,9 +48,18 @@ func (s *Server) handleLaunch(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "missing iss in URL query parameters", http.StatusUnauthorized)
 		return
 	}
-	if iss[0] != s.AuthorizedFHIRURL {
+	if iss[0] != s.authorizedFHIRURL {
 		http.Error(w, fmt.Sprintf("unauthorized iss %s", iss[0]), http.StatusUnauthorized)
 		return
 	}
+
+	sess, err := s.sm.New(w, r)
```

Comment:
should we try to find existing session first? Also, we should probably only create a new session once user logged in on the EHR server?

Reply:
Add a TODO here for finding the existing session first.
For the second question:
session is critical in the middle of FHIR authentication - we want to store FHIR URL and state in session.

Reply:
ack